### ISSUE TITLE
Issue on Autocrop Website

### ISSUE THREAD

**Issue by rigard** [tmb5cg/tmb5cg#1]
Hi. There doesn't seem to be a way to contact you. I created an account and paid for it. Unfortunately, my mailserver was down at the time and I didn't receive a login code. Trying to use the forgot password function outputs an error due to the current state of my account. Could you look into this, please?

**Comment by tmb5cg** [comment]
@rigard  yes I am sorry about this! I'm looking into it now, you should get an email shortly.

**Comment by tmb5cg** [comment]
Just sent you an email
